src/components/Captcha/index.js

```diff
@@ -0,0 +1,72 @@
+import PropTypes from 'prop-types'
+import React, { useState, useRef, useEffect } from 'react'
+import Reaptcha from 'reaptcha'
+
+const Captcha = ({
+    captchaTheme,
+    errors,
+    register,
+    setValue,
+    wrapClassName,
+}) => {
+    if (!process.env.RECAPTCHA_SITE_KEY) {
```

Comment:
```suggestion
    if (!process.env.GATSBY_RECAPTCHA_SITE_KEY) {
```

env variables not prefixed with `GATSBY_` won't be available on the client

SEE: https://www.gatsbyjs.org/docs/environment-variables/#client-side-javascript